ui/src/main/webapp/src/app/project/migration-project-form.component.html

```diff
@@ -1,6 +1,23 @@
 <div class="container-fluid wu-frame">
     <form #projectForm="ngForm" (ngSubmit)="save()" class="form-horizontal wizard-form">
 
+        <style>
```

Comment:
:+1: To @jsight's comment. We need some consistency, so it doesn't make any sense to use default patternfly styles everywhere else and just here (for reason which I don't really fully understand) use something different.